src/main/java/Wolox/training/models/Book.java

```diff
@@ -0,0 +1,120 @@
+package Wolox.training.models;
+
+import javax.persistence.*;
+
+@Entity
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(nullable = false, unique = true)
+    private String title;
+
+    @Column(nullable = false)
+    private String author;
+
+    private String gender;
+
+    @Column(nullable = false)
+    private String image;
+
+    @Column(nullable = false)
+    private String subtitle;
+
+    @Column(nullable = false)
+    private String publisher;
+
+    @Column(nullable = false)
+    private String year;
+
+    @Column(nullable = false)
+    private Integer pages;
+
+    @Column(nullable = false)
+    private String isbn;
+
+    public Book(String title, String author, String gender, String image, String subtitle, String publisher, String year, Integer pages, String isbn) {
```

Comment:
Here is missing the default non args constructor that is required JPA
```suggestion

   public Book() {}

    public Book(String title, String author, String gender, String image, String subtitle, String publisher, String year, Integer pages, String isbn) {
```